### ISSUE TITLE
Make using `layout` for arbitrary components more discoverable

### ISSUE THREAD

**Issue by cwickham** [quarto-dev/quarto-cli#6192]
Using `layout` to arrange things other than figures is possible but currently examples are buried in [Guide > Authoring > Figures](https://quarto.org/docs/authoring/figures.html#block-layout) and [Guide > Interactivity > Component Layout](https://quarto.org/docs/interactive/layout.html#panel-layout).

Perhaps it should be discussed as part of [Guide > Authoring > Article Layout](https://quarto.org/docs/authoring/article-layout.html)

**Comment by mcanouil** [comment]
I realise that I talked about this last summer with the Quarto team and was hoping to add the "layout" example close to the multi-columns examples, _e.g._ <https://quarto.org/docs/presentations/index.html#multiple-columns>. 😅

For example, 

```
:::: {layout="[ 40, 60 ]"}

::: {#first-column}
structured

contents...
:::

::: {#second-column}
structured

contents...
:::

::::
```

**Comment by cwickham** [comment]
Yeah, I have that as a todo for [this issue](https://github.com/quarto-dev/quarto-cli/issues/4600)
Currently plan to add something like that to [Block Layout](https://quarto.org/docs/authoring/figures.html#block-layout), then this issue can address if that section should live somewhere else on the site.

**Comment by cwickham** [comment]
Also a reminder to make sure wherever this ends up it is discovered when people search for things like "Multiple Columns".
